src/nav/browser.yml

```
@@ -35,6 +33,14 @@ pages:
         path: /docs/browser/new-relic-browser/configuration/rename-browser-apps
       - title: Delete browser apps
         path: /docs/browser/new-relic-browser/installation/delete-apps-new-relic-browser
+  - title: Guides
+    pages: 
+      - title: Browser best practice guide
```

Or this to parallel the others
```suggestion
      - title: Guide: Monitoring browser apps 
```